tests/fixtures/data.yml

```diff
@@ -9,12 +9,16 @@ person:
     ORIGINAL_HIRE_DATE: !!timestamp 2001-01-01 00:00:00.0
     APPOINTMENT_END_DATE: !!timestamp 2010-01-01 00:00:00.0
     JOB_TITLE: Adjunct Professor
-    PERSONNEL_SUBAREA_CODE: CFAC
-    HR_ORG_UNIT_TITLE: Chemisty
+    PERSONNEL_SUBAREA_CODE: CFAT
+    HR_ORG_UNIT_ID: "1"
+    HR_ORG_LEVEL4_NAME: Chemistry
```

Comment:
I may be misunderstanding what you're worried about here, but the cli test does load the data up and do a real query against it. If you change the value of HR_ORG_LEVEL4_NAME in data.yml you'll see it fail.